pkg/pipelines/bootstrap_test.go

```diff
@@ -120,8 +120,7 @@ func TestBootstrapManifest(t *testing.T) {
 		"05-pipelines/app-ci-pipeline.yaml",
 		"05-pipelines/cd-deploy-from-push-pipeline.yaml",
 		"05-pipelines/ci-dryrun-from-pr-pipeline.yaml",
-		"06-bindings/github-pr-binding.yaml",
-		"06-bindings/github-push-binding.yaml",
+		"06-bindings/github-binding.yaml",
```

Comment:
I think we should retain the `github-<event>-binding` nomenclature here.

So, I'd have `github-push-binding.yaml`.

Reply:
I've retained the naming convention